src/test/ui/let-else/let-else-drop-order.run.stdout

```diff
@@ -0,0 +1,32 @@
+v is 0
+  if:
+    I dropped
+    if body
+  if let:
+    if body
+    I dropped
+  if let chain with let:
+    if body
+    I dropped
+  if let chain with bool:
+    I dropped
+    if body
+  let else:
+    I dropped
+    if body
+v is 1
```

Comment:
Correct me if I'm wrong, but currently *none* of these cases "differ" between the `then` case and then `else` case, right? In the sense that if you put these two halves side-by-side, the "I dropped" would all be aligned.

I'm wondering whether there's any value in revising the presentation to make that immediate, by actually have the textual output *be* two columns...

Reply:
> currently none of these cases "differ" between the then case and then else case, right?

There were differences back when I filed my PR. I've just checked out a nightly from july 15 when I filed it, and got this:

```
  let else:
    else body
    I dropped
```
But now you are correct, it's all the same in the match and mismatch case.

As for changes in the layout, I really like how this (now outdated) table turned out: https://github.com/rust-lang/rust/issues/87335#issuecomment-1186382364 . side by side view is really nice indeed :)